app/Http/Controllers/Users.php

```diff
@@ -12,37 +12,85 @@
 
 class Users extends Controller
 {
+    /**
+     * @var Request $request;
+     */
+    private $request;
+
+    /**
+     * @var Redirect $redirect;
+     */
+    private $redirect;
+
+    /**
+     * @var Session $session;
+     */
+    private $session;
+
+    /**
+     * @var Log $logger;
+     */
+    private $logger;
+
+    /**
+     * @var Database $database;
+     */
+    private $database;
+
+    /**
+     * @var CreateUsersRequest $usersRequest;
+     */
+    private $usersRequest;
+
+    /**
+     * Users Constructor
+     *
+     * @param Request $request
+     * @param Redirect $redirect
+     * @param Session $session
+     * @param Log $logger
+     * @param Database $database
+     * @param CreateUsersRequest $usersRequest
+     */
+    public function __construct(Request $request, Redirect $redirect, Session $session, Log $logger, Database $database, CreateUserRequest $usersRequest)
```

Comment:
Since the line is too long, this should be formatted like this according to PSR

    public function __construct(
        Request $request,
        Redirect $redirect,
        Session $session,
        Log $logger,
        Database $database,
        CreateUserRequest $usersRequest
    ) {
        $this->request = $request;
        $this->redirect = $redirect;
        $this->session = $session;
        $this->logger = $logger;
        $this->database = $database;
        $this->usersRequest = $usersRequest;
    }